xm-commons-lep-groovy/src/main/java/com/icthh/xm/commons/lep/TenantScriptStorage.java

```diff
@@ -2,6 +2,6 @@
 
 public enum TenantScriptStorage {
 
-    CLASSPATH, XM_MS_CONFIG, FILE;
+    CLASSPATH, XM_MS_CONFIG, FILE, XM_MS_CONFIG_DIRTY_CLASSPATH;
```

Comment:
how to use FILE  + XM_MS_CONFIG_DIRTY_CLASSPATH ?

Reply:
FILE it`s already have "dirty classpath" and no import-s cache